tlscanary/template/js/report_page.js

```diff
@@ -195,44 +184,43 @@ function makeMetaTab(meta)
     html += "</tbody>";
   }
   html += "</table>";
+
+  const element = window.document.getElementById("metadata");
   element.innerHTML = html;
 }
 
-function navigate(tab)
-{
-  var $nav = $("#nav");
-  var listItems = $nav.children();
-  var tabs = ["results", "chart", "metadata"];
-  for (var i=0;i<tabs.length;i++)
+function navigate(tab) {
+  const $nav = $("#nav");
```

Comment:
I have missed this before, but what is the thinking behind leading variable names with `$`?

Reply:
This comes from jQuery, which we get for free because we are using `bootgrid`. It provides easier ways to look up DOM elements. I have used it in a few places when it was simpler than using a pure JS solution. If consistency is important, I could rewrite all DOM lookups to be either pure JS or jQuery.

Reply:
Alright, thanks for clarifying. All fine if this was intentional.